euphonic/qpoint_phonon_modes.py

```diff
@@ -309,18 +317,20 @@ def calculate_debye_waller(
 
         .. math::
 
-          e^{-W} = e^{-\\sum_{\\alpha\\beta}{W^{\\kappa}_{\\alpha\\beta}Q_{\\alpha}Q_{\\beta}}}
+          e^{-W} = e^{-\\sum\\limits_{\\alpha\\beta}{W^{\\kappa}_{\\alpha\\beta}Q_{\\alpha}Q_{\\beta}}}
 
         The Debye-Waller exponent is defined as
         :math:`W^{\\kappa}_{\\alpha\\beta}` and is independent of Q, so
         for efficiency can be precalculated to be used in the structure
-        factor calculation. The Debye-Waller exponent is calculated by [2]_
+        factor calculation. The Debye-Waller exponent is calculated by [2]_.
+        Note that internally Euphonic uses atomic units so :math:`\\hbar`
+        has been omitted from the formulation:
 
         .. math::
 
           W^{\\kappa}_{\\alpha\\beta} =
-          \\frac{1}{4M_{\\kappa}\\sum_{q}{weight_q}}
-          \\sum_{q\\nu}weight_q\\frac{\\epsilon_{q\\nu\\kappa\\alpha}\\epsilon^{*}_{q\\nu\\kappa\\beta}}
+          \\frac{1}{4M_{\\kappa}\\sum\\limits_{q}{weight_q}}
+          \\sum_\\limits{q\\nu}weight_q\\frac{\\epsilon_{q\\nu\\kappa\\alpha}\\epsilon^{*}_{q\\nu\\kappa\\beta}}
```

Comment:
Have you got the subscript _ in the right place? Previous lines have \sum\limits_.

Reply:
Good spot, corrected.